roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -13,54 +13,65 @@
     group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   loop:
-    - {"path": "/var/lib/openstack/config/libvirt"}
-    - {"path": "/var/lib/openstack/config/containers"}
-    - {'path': "/var/lib/nova"}
     - {"path": "/etc/tmpfiles.d/", "owner": "root", "group": "root"}
     - {"path": "/var/lib/edpm-config/firewall", "owner": "root", "group": "root"}
 
-- name: Create container config dirs
-  become: true
+- name: Render libvirt config files
   tags:
     - configure
     - libvirt
-  ansible.builtin.file:
-    path: "/var/lib/openstack/config/libvirt/{{ item }}"
-    state: directory
-    setype: "{{ item.setype | default('container_file_t') }}"
-    owner: "{{ item.owner | default(ansible_user) }}"
-    group: "{{ item.group | default(ansible_user) }}"
-    mode: "0755"
-  loop: "{{ edpm_libvirt_containers }}"
+  become: true
+  ansible.builtin.template:
+    src: "{{ item.src }}"
+    dest: "/etc/libvirt/{{ item.dest }}"
+    mode: "0644"
```

Comment:
it shouldn't be world read

Reply:
[root@edpm-compute-0 ~]# ls -alZ /etc/libvirt/
total 144
drwx------.   6 root root system_u:object_r:virt_etc_t:s0      4096 Nov 21 12:35 .
drwxr-xr-x. 114 root root system_u:object_r:etc_t:s0           8192 Nov 27 12:34 ..
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       450 Jul  3 10:08 libvirt-admin.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       547 Jul  3 10:08 libvirt.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0     17826 Aug 25 08:54 libvirtd.conf
drwx------.   2 root root system_u:object_r:virt_etc_rw_t:s0   4096 Nov 16 14:32 nwfilter
drwx------.   4 root root system_u:object_r:virt_etc_rw_t:s0     39 Nov 23 09:10 qemu
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       242 Nov 16 14:33 qemu.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0      2169 Aug 25 08:54 qemu-lockd.conf
drwx------.   2 root root system_u:object_r:virt_etc_rw_t:s0      6 Aug 25 08:55 secrets
drwx------.   3 root root system_u:object_r:virt_etc_rw_t:s0     23 Nov 16 14:32 storage
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0     12421 Aug 25 08:54 virtinterfaced.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0      3058 Jul  3 10:08 virtlockd.conf
-rw-r--r--.   1 root root system_u:object_r:virtlogd_etc_t:s0   106 Nov 16 14:33 virtlogd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0     12411 Aug 25 08:54 virtnetworkd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       235 Nov 16 14:33 virtnodedevd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0     12416 Aug 25 08:54 virtnwfilterd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       259 Nov 16 14:33 virtproxyd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       232 Nov 21 12:35 virtqemud.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0       234 Nov 16 14:33 virtsecretd.conf
-rw-r--r--.   1 root root system_u:object_r:virt_etc_t:s0     12411 Aug 25 08:54 virtstoraged.conf


even the files that come form the rpm are 644 by default just an fyi
i do not modify qemu-lockd.conf or libvirtd.conf  above so they have the rpm defaualt content and mode.

i can drop the world read but 0644 is the expected mode for files in /etc/ in general

ill update it to make this 0640 but that will be a change in behavior form how this is normally done.